novawallet/Modules/BackupAttention/CheckboxListPresenter/CheckboxListViewModelFactory.swift

```diff
@@ -0,0 +1,82 @@
+import UIKit
+import SoraFoundation
+
+class CheckboxListViewModelFactory {
+    let localizationManager: LocalizationManagerProtocol
+
+    init(localizationManager: LocalizationManagerProtocol) {
+        self.localizationManager = localizationManager
+    }
+
+    // swiftlint:disable function_body_length
```

Comment:
I would move individual item of the list into a separate. It will improve readability and stop SwiftLint complains